src/main/scala/uk/gov/hmrc/play/it/MicroServiceEmbeddedServer.scala

```diff
@@ -70,65 +66,6 @@ trait ExternalServiceOrchestrator extends StartAndStopServer {
   }
 }
 
-trait EmbeddedServiceOrchestrator extends ResourceProvider with StartAndStopServer {
```

Comment:
The code below contains functionality that is specific to service manager. Some generic ScalaTest/Play support cannot possibly contain matching functionality. The below is essential for running it tests, so I'm afraid we cannot simply remove it.